distribution/docs/src/main/resources/_contents/_developing/developing-intro-contents.adoc

```diff
@@ -0,0 +1,816 @@
+
+== ${branding} Development Guidelines
+
+[NOTE]
+====
+Development requires full knowledge of the ${branding} Catalog.
+====
+
+${branding} is written in Java and requires a moderate amount of experience with the Java programming language, along with Java terminology, such as packages, methods, classes, and interfaces.
+${branding} uses a small OSGi runtime to deploy components and applications.
+Before developing for ${branding}, it is necessary that developers have general knowledge on OSGi and the concepts used within.
+This includes, but is not limited to, Catalog Commands and the following topics:
+
+* The Service Registry
+** How services are registered
+** How to retrieve service references
+* Bundles
+** Their role in OSGi
+** How they are developed
+
+Documentation on OSGi can be viewed at the http://www.osgi.org[OSGi Alliance website].
+Helpful literature for beginners includes _OSGi and Apache Felix 3.0 Beginner's Guide_ by Walid Joseph Gédéon and _OSGi in Action: Creating Modular Applications in Java_ by Richard Hall, Karl Pauls, Stuart McCulloch, and David Savage.
+
+=== Recommended Hardware
+
+Because of its modular nature, ${branding} may require a few or many system resources, depending on which bundles and features are deployed.
+In general, ${branding} will take advantage of available memory and processors.
+A 64-bit JVM is required, and a typical installation is performed on a single machine with 16GB of memory and eight processor cores.
+
+The ${branding} source code is not tied to any particular IDE.
+However, if a developer is interested in setting up the Eclipse IDE, they can view the http://books.sonatype.com/m2eclipse-book/reference/[Sonatype guide] on developing with Eclipse.
+
+==== Getting Set Up
+
+To develop on ${branding}, access to the source code via Github is required.
+
+===== Integrated Development Environments (IDE)
+
+The ${branding} source code is not tied to any particular IDE.
+However, if a developer is interested in setting up the Eclipse IDE, they can view the http://books.sonatype.com/m2eclipse-book/reference/[Sonatype guide] on developing with Eclipse.
```

Comment:
We might (really might) want to add a section here on Intellij as well (for reasons that will become clear with my next comment)...There is a community edition that is free.